rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/interpreter/internal_interpreter.kt

```diff
@@ -1389,17 +1392,15 @@ class InternalInterpreter(val systemInterface: SystemInterface) : InterpreterCor
                     return StringValue(sourceString.replaceRange(0..replStringLength - 1, replString))
                 }
                 // case of %REPLACE(stringToReplaceWith:stringSource:startIndex)
-                if (expression.start != null && expression.length == null) {
+                if (expression.length == null) {
```

Comment:
Isn't a risk start being null?

Reply:
The previous lines are
`               if (expression.start == null) {
                    return StringValue(sourceString.replaceRange(0..replStringLength - 1, replString))
                }`
So the compiler suggests to remove the null check